ui/components/component-library/avatar-network/avatar-network.types.ts

```diff
@@ -0,0 +1,55 @@
+import { AvatarBaseSize } from '../avatar-base/avatar-base.types';
+import { ValidTag } from '../text/text.types';
+import type { BoxProps } from '../../ui/box/box.d';
+import {
+  BackgroundColor,
+  BorderColor,
+  TextColor,
+} from 'ui/helpers/constants/design-system';
+
+/**
+ * Props for the AvatarNetwork component
+ */
+export interface AvatarNetworkProps extends BoxProps {
+  /**
+   * The name accepts the string to render the first alphabet of the Avatar Name
+   */
+  name?: string;
+  /**
+   * The src accepts the string of the image to be rendered
+   */
+  src?: string;
+  /**
+   * The showHalo accepts a boolean prop to render the image with halo effect
+   */
+  showHalo?: boolean;
+  /**
+   * The size of the AvatarNetwork
+   * Possible values could be AvatarBaseSize.Xs(16px), AvatarBaseSize.Sm(24px), AvatarBaseSize.Md(32px), AvatarBaseSize.Lg(40px), AvatarBaseSize.Xl(48px)
+   * Defaults to AvatarBaseSize.Md
+   */
+  size?: AvatarBaseSize;
```

Comment:
Is there a way we can still create the `AvatarNetworkSize` type but allow AvatarBase to accept it?

Reply:
One way is to allow a "size" property of AvatarBaseProps to accept both AvatarBaseSize | AvatarNetworkSize.